### ISSUE TITLE
ch-remote: panic in main when no command is given

### ISSUE THREAD

**Issue by matdibu** [cloud-hypervisor/cloud-hypervisor#6409]
**Describe the bug**
`ch-remote` crashes when invoked without a command

**To Reproduce**
Steps to reproduce the behaviour:
invoke `ch-remote` with `--api-socket` and a valid path, but without a comand:
```bash
$ ch-remote --api-socket /var/run/cloud-hypervisor.socket
thread 'main' panicked at src/bin/ch-remote.rs:509:14:
internal error: entered unreachable code
stack backtrace:
   0:     0x555fb8b83995 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h83d3e31f1809523c
   1:     0x555fb8b65f80 - core::fmt::write::h76573a5842288faa
   2:     0x555fb8b96b84 - std::io::Write::write_fmt::hb095d534967ab243
   3:     0x555fb8b84198 - std::sys_common::backtrace::print::h73765b495122298a
   4:     0x555fb8b840e0 - std::panicking::default_hook::{{closure}}::hfad133b7877f7d1a
   5:     0x555fb8b847b6 - std::panicking::rust_panic_with_hook::h03308f3977d70e82
   6:     0x555fb8b84358 - std::panicking::begin_panic_handler::{{closure}}::h6543c8dd74f459e9
   7:     0x555fb8b842e6 - std::sys_common::backtrace::__rust_end_short_backtrace::h68617e661e847078
   8:     0x555fb8b842d3 - rust_begin_unwind
   9:     0x555fb8b1b7c4 - core::panicking::panic_fmt::hf7f647d933c4266a
  10:     0x555fb8b1b962 - core::panicking::panic::h423064a40da58821
  11:     0x555fb8b24103 - ch_remote::TargetApi::do_command::hf8301c074d50aaa2
  12:     0x555fb8b2bb4c - ch_remote::main::h0828bc8106cb292d
  13:     0x555fb8b1ffb3 - std::sys_common::backtrace::__rust_begin_short_backtrace::h163e44049b360cc3
  14:     0x555fb8b2ce3d - main
  15:     0x7f5d3fd5510e - __libc_start_call_main
  16:     0x7f5d3fd551c9 - __libc_start_main@@GLIBC_2.34
  17:     0x555fb8b1ef15 - _start
  18:                0x0 - <unknown>
```

**Version**

Output of `cloud-hypervisor --version`: `cloud-hypervisor v38.0.0`

Did you build from source, if so build command line (e.g. features):

**VM configuration**

What command line did you run (or JSON config data):

Guest OS version details:

Host OS version details:

**Logs**

Output of `cloud-hypervisor -v` from either standard error or via `--log-file`:

Linux kernel output:

**Comment by matdibu** [comment]
nevermind, looks like intended behaviour: https://github.com/cloud-hypervisor/cloud-hypervisor/blob/e6aa57e3b90458212f91b4c232860e332b9bd458/src/bin/ch-remote.rs#L510C25-L510C29

I don't know rust, but I didn't expect to see a main thread panic when parsing arguments.

**Comment by rbradford** [comment]
I think it might be nicer to have a proper error message.

**Comment by liuw** [comment]
This has been fixed in the latest upstream code in 5627c26405399a511af1eadb3a3ffa4ac8c1cde0.
